src/richie/apps/courses/serializers.py

```diff
@@ -75,6 +75,8 @@ class Meta:
             "state",
             "enrollment_count",
             "catalog_visibility",
+            "offer",
+            "price",
```

Comment:
`price` field could be returned as a float not a string.

Reply:
I've made a change to this, but I'm not sure if it's in line with what you asked for.